### ISSUE TITLE
Why do we have a makefile

### ISSUE THREAD

**Issue by stevenruidigao** [CryptoChat-dev/cli#13]
Firstly, why do we have a makefile?
Secondly, we forgot to tell people to install nodejs.

**Comment by ghost** [comment]
The makefile is there to facilitate a uniform install process across all platforms supporting make. 

EDIT: nobody wants to have to `cd cli && node cli.js` to use an app: the shellscript and makefile solve this.

**Comment by stevenruidigao** [comment]
> The makefile is there to facilitate a uniform install process across all platforms supporting make.
> 
> EDIT: nobody wants to have to `cd cli && node cli.js` to use an app: the shellscript and makefile solve this.

use regular shellscripts then
why use makefiles?

**Comment by ghost** [comment]
The makefile moves the shellscript/launcher to /usr/local/bin, which is in your $PATH, and moves the actual js and node modules to /var/lib/cryptochat. this is meant to clean up usage of the app, only requiring the user to interact with the shellscript. did you read the makefile or are you just complaining that it's there?

**Comment by stevenruidigao** [comment]
Why aren't we just using a shellscript to install instead of a makefile?

**Comment by ghost** [comment]
what's *wrong* with using make?

**Comment by stevenruidigao** [comment]
There's no point, and using regular shell scripts is a lot simpler?

**Comment by ghost** [comment]
Simpler? Not really. Pointless? Maybe, but it works fine.

**Comment by stevenruidigao** [comment]
also, btw here's a bundled version for amd64 (linux): https://filen.io/d/126874fd-f831-4de9-9791-9e26ae2c9f65#!vCLxQytP6b1CNJbYYpkrvfIERYbB1bua

**Comment by stevenruidigao** [comment]
> Simpler? Not really. Pointless? Maybe, but it works fine.

I mean, it at least would remove the dependency on make, but idk.
make is mostly designed for compiling programs, but w/e.

**Comment by ghost** [comment]
yeah that's the only thing i can think of, and it's a good point. maybe i can make an interactive installer in bash?
